src/gen_fn_interface.jl

```diff
@@ -69,14 +69,22 @@ Return the generative function that produced the given trace.
 function get_gen_fn end
 
 """
-    value = trace[addr]
+    value = getindex(trace::Trace, addr)
 
-Get the value of the random choice at address `addr`.
+Get the value of the random choice, or auxiliary state (e.g. return value of inner function call), at address `addr`.
 """
 function Base.getindex(trace::Trace, addr)
-    get_choices(trace)[addr]
+    error("Not implemented")
```

Comment:
I agree with @alex-lew that this should be kept. I don't see a need to remove it. Returning the choice at a given address if there is one, is indeed always the correct behavior for `getindex`. Do you agree @bzinberg ?

Reply:
Yeah, I agree it's fine to keep, given that `getindex` is the only accessor
for aux data.  If there were a `get_aux` method in the GFI I would have
wanted to use that in the generic `getindex`. +1